test/test-container-start-command.js

```diff
@@ -6,14 +6,16 @@
 var Container = require('../lib/drivers/direct/container');
 var tap = require('tap');
 
-var basicOpts = {
-  instanceId: '_id',
-  baseDir: '_base',
-  server: {},
-};
+function newContainer() {
+  return new Container({
+    instanceId: '_id',
+    baseDir: '_base',
+    server: {},
```

Comment:
Will these parameters ever change? Not sure of deeper functionality so can't say if this needs an `options`.